translations/store-translations/malte.json

```diff
@@ -15,6 +15,9 @@
     "fr": {
       "keywords": "Allemagne,Malteser,aide,apprendre l'allemand,Règles,Cours d'allemand,Nouvelles,Conseil,Intégration"
     },
+    "hr": {
+      "keywords": "Malteser,Deutschland,Hilfe,Deutsch lernen,Regeln,Deutschkurs,Nachrichten,Beratung,Informationen,Asyl"
```

Comment:
So we should not forget to receive the croation translations next time?

Reply:
Yes, on my list.